Zotero/Models/Tag.swift

```diff
@@ -11,25 +11,29 @@ import UIKit
 struct Tag: Identifiable, Equatable, Hashable {
     let name: String
     let color: String
+    let emojiGroup: String?
     let type: RTypedTag.Kind
 
     var id: String { return self.name }
 
-    init(name: String, color: String) {
+    init(name: String, color: String, emojiGroup: String?) {
```

Comment:
A convenience `init` could be added that passes then `EmojiExtractor.extractFirstContiguousGroup(from: name)` to the regular `init`.

Reply:
Actually I'll just get rid of `emojiGroup` and include the `EmojiExtractor` in initializer. This initializer is used in one place only anyway.